### ISSUE TITLE
JSONObject has fields unknown to BigQuery: root.database.

### ISSUE THREAD

**Issue by redcricket** [zendesk/maxwell#1995]
I have been able to get maxwell to connect to GCP BigQuery.

```
cd maxwell-1.40.0/bin
export GOOGLE_APPLICATION_CREDENTIALS=/home/red/mydbprj001-468b7f03ed35.json
./maxwell --user=maxwell --password=$PW --producer=bigquery \
--filter='exclude: my_config.ri_states, exclude: my_reporting.*' \
--log_level=debug \
--bigquery_project_id=mydbprj001 \
--bigquery_dataset=new_data_set_01 \
--bigquery_table=examples
```

Then on MySQL server I will update the examples table like this:

```
MariaDB [my_config]> update examples set notes = 'We will make an example of you!' where name = 'Example Inc';
```

Then maxwell crashes.

```
40586 [DEBUG] MysqlPositionStore: Writing binlog position to maxwell.positions: Position[BinlogPosition[0-0-1864034], lastHeartbeat=1680713765530], last heartbeat read: 1680713765530
40786 [DEBUG] C3P0PooledConnectionPool: Testing PooledConnection [com.mchange.v2.c3p0.impl.NewPooledConnection@3e011b82] on CHECKOUT.
40787 [DEBUG] C3P0PooledConnectionPool: Test of PooledConnection [com.mchange.v2.c3p0.impl.NewPooledConnection@3e011b82] on CHECKOUT has SUCCEEDED.
41536 [INFO] TaskManager: Stopping 3 tasks
41537 [ERROR] TaskManager: cause:
java.lang.IllegalArgumentException: JSONObject has fields unknown to BigQuery: root.database.
        at com.google.cloud.bigquery.storage.v1.JsonToProtoMessage.convertJsonToProtoMessageImpl(JsonToProtoMessage.java:205) ~[google-cloud-bigquerystorage-2.14.2.jar:2.14.2]
        at com.google.cloud.bigquery.storage.v1.JsonToProtoMessage.convertJsonToProtoMessage(JsonToProtoMessage.java:164) ~[google-cloud-bigquerystorage-2.14.2.jar:2.14.2]
        at com.google.cloud.bigquery.storage.v1.JsonStreamWriter.append(JsonStreamWriter.java:147) ~[google-cloud-bigquerystorage-2.14.2.jar:2.14.2]
        at com.google.cloud.bigquery.storage.v1.JsonStreamWriter.append(JsonStreamWriter.java:106) ~[google-cloud-bigquerystorage-2.14.2.jar:2.14.2]
        at com.zendesk.maxwell.producer.MaxwellBigQueryProducerWorker.sendAsync(MaxwellBigQueryProducer.java:275) ~[maxwell-1.39.6.jar:1.39.6]
        at com.zendesk.maxwell.producer.AbstractAsyncProducer.push(AbstractAsyncProducer.java:93) ~[maxwell-1.39.6.jar:1.39.6]
        at com.zendesk.maxwell.producer.MaxwellBigQueryProducerWorker.run(MaxwellBigQueryProducer.java:252) ~[maxwell-1.39.6.jar:1.39.6]
        at java.lang.Thread.run(Thread.java:829) [?:?]
```

I have the BigQuery table set up correctly (maybe).

I created it the the `bq` command line like so ...

```
[red.cricket@demo-dmz1 MAXWELL]$ bq load --source_format=CSV --autodetect new_data_set_01.example examplechange_dot_data.csv
Upload complete.
Waiting on bqjob_r10d9c0c53ea02d8e_0000018752568093_1 ... (1s) Current status: DONE
```

Maybe this is an issue, or maybe I just don't know what I am doing.  Any help is appreciated.

**Comment by redcricket** [comment]
Never mind.  I did not set up my BigQuery table up correctly.  I had not understood the documentation I found here: https://maxwells-daemon.io/producers/#google-cloud-bigquery.  For anyone else that doesn't understand that bit of documentation.  This is what I did to fix my BigQuery table.

export SCHEMA=database:string,table:string,type:string,ts:integer,xid:integer,xoffset:integer,commit:boolean,position:string,gtid:string,server_id:integer,primary_key:string,data:string,old:string
bq load --source_format=CSV new_data_set_01.tenants schema.sql $SCHEMA

Sorry for opening a non-issue.
